### PR TITLE
Fix for issue #4907

### LINKED ISSUE #4907: Right part of top icons seems to be positioned too far right in 5.0 if window is small
JabRef version 5.0-dev (master--latest)

- [ x] I have tested the latest development version from http://builds.jabref.org/master/ and the problem persists

This concerns the top icons in JabRef, like new database, open database, save, etc (right below the menu). The right portion of those icons, including new article, delete article, undo, redo, cut, etc, seems to be positioned too far right if there is limited space available (e.g. window being resized to being somewhat small). In those cases there is a lot of space available in that area, but the icons are displayed out of the screen and the small `>>` arrows button is shown, which on click shows those icons. The search bar is displayed like in the middle of the available space it seems.

My system setup, should this be a system specific thing:
Arch Linux, i3 window manager, java version:
```
java -version
java version "1.8.0_201"
Java(TM) SE Runtime Environment (build 1.8.0_201-b09)
Java HotSpot(TM) 64-Bit Server VM (build 25.201-b09, mixed mode)
```

1. Resize Jabref to a smaller size
2. Observe the icons on top: the left portion is displayed correctly, the right displayed too far right, despite there is still space available to show buttons there.

2019-04-22-151910_1295x527_scrot2

### PR DESCRIPTION
**Pull request by core-master**
Fixes #4907 

About the toolbar, I don't think the search bar size should be link to side pane size with a Bind. It's not really responsive :

littleBefore

bigBefore

-----
Here without the bind :

bigAfter

littleAfter

----

- [ ] Change in CHANGELOG.md described
- [ ] Tests created for changes
- [X] Manually tested changed features in running JabRef
- [X] Screenshots added in PR description (for bigger UI changes)
- [X] Ensured that [the git commit message is a good one](https://github.com/joelparkerhenderson/git_commit_message)
- [X] Check documentation status (Issue created for outdated help page at [help.jabref.org](https://github.com/JabRef/help.jabref.org/issues)?)

### CODE REVIEW

**Comment by Siedlerchr** [comment]
I like the idea! That's one thing always bothered me

File: src/main/java/org/jabref/gui/JabRefFrame.java [hunk 1]
```diff
@@ -519,8 +519,9 @@ private Node createToolbar() {
                 leftSpacer
         );
         leftSide.setMinWidth(100);
-        leftSide.prefWidthProperty().bind(sidePane.widthProperty());
-        leftSide.maxWidthProperty().bind(sidePane.widthProperty());
+        // Probably useless bind
+        /*leftSide.prefWidthProperty().bind(sidePane.widthProperty());
```
Comment (LinusDietz) [inline src/main/java/org/jabref/gui/JabRefFrame.java]: If it is useless, delete it.

**Comment by tobiasdiez** [comment]
Please add at least a bit of space between the search bar and the icons. Maybe try to center the search bar in the toolbar and see how it looks.

**Comment by Siedlerchr** [comment]
How is the status of this PR? It would be really helpful if you could fix the little issues mentioned here, so we can merge this PR.

**Pull request closed without merging**